README.md

```diff
@@ -122,7 +122,7 @@ During development, Dockerfile frontend (dockerfile.v0) is also part of the Buil
 
 ```
 buildctl build --frontend=dockerfile.v0 --local context=. --local dockerfile=.
-buildctl build --frontend=dockerfile.v0 --local context=. --local dockerfile=. --frontend-opt target=foo --frontend-opt build-arg:foo=bar
+buildctl build --frontend=dockerfile.v0 --local context=. --local dockerfile=. --opt target=foo --opt build-arg:foo=bar
```

Comment:
Bit on the fence on the `--frontend-opt` -> `--opt` change, as it's not clear to me that they are related to each other. 😅

Reply:
Why should you as a user care if they are related or not?

Reply:
To understand that an option is related to the frontend, not to (e.g.) the `--output`.

If we drop the prefix, should we also drop the `--export-` prefix from `--export-cache`?

Reply:
I wouldn't say they are strictly related to the frontend. They are more like options for the build request, while the other options are for a specific component (exporter, cache) or are cli side (local, secret).

Reply:
Ah; gotcha - guess I got confused because they previously were prefixed